composio/local_tools/local_workspace/cmd_manager/actions/search_cmds.py

```diff
@@ -40,7 +40,7 @@ class SearchDirCmd(BaseAction):
     @history_recorder()
     def execute(
         self, request_data: SearchDirRequest, authorisation_data: dict
-    ) -> SearchDirResponse:
+    ) -> BaseResponse:
```

Comment:
Why was the output changed to BaseResponse?

Reply:
needed to wrap outputs with history and state of the workspace --> its needed to keep all response in baseresponse class